StRoot/StMuDSTMaker/COMMON/StMuFwdTrack.h

```diff
@@ -121,6 +121,8 @@ class StMuFwdTrack : public TObject {
 
     // Number of points used in the track seed step
     short   numberOfSeedPoints() const;
+    UShort_t idTruth() const { return mIdTruth; }
+    UShort_t qaTruth() const { return mQATruth; }
 
 
     void setPrimaryMomentum( TVector3 mom ) { mPrimaryMomentum = mom; }
```

Comment:
I suggest switching to StThreeVector for speed.  TVector3 has an additional overhead b/c it inherits from TObject.  Which means that every instance of every vector you create will be registered in a global ROOT map, and subsequently destroyed when you call the destructor.  

Looking at the MuDST codes... I see that we are using both TVector3 and StThreeVector<x>  (with x=float or double) to store vector quantities.... so consider this a soft suggestion.